kommunicate/src/main/java/io/kommunicate/KmConversationHelper.java

```diff
@@ -732,6 +723,15 @@ public void onSuccess(Object message) {
                     if (useSingleThreadedSettingFromServer) {
                         conversationBuilder.setSingleConversation(kmAppSettings.getChatWidget().isSingleThreaded());
                     }
+                    if(kmAppSettings.getChatWidget().getZendeskChatSdkKey() != null) {
+                        if (conversationBuilder.getConversationInfo() != null) {
+                            conversationBuilder.getConversationInfo().put(KmConstants.CONVERSATION_SOURCE, KmConstants.ZOPIM);
```

Comment:
move all this logic to conversationbuilder where conversation info map code is present.
from here, the code should be something like:

conversationBuilder.updateConversationInfo(KmConstants.CONVERSATION_SOURCE, KmConstants.ZOPIM);